gitclone/checkout.go

```diff
@@ -217,7 +291,7 @@ func selectFallbacks(checkoutStrategy CheckoutMethod, fetchOpts fetchOptions) fa
 		return nil
 	case CheckoutCommitMethod, CheckoutTagMethod:
 		return simpleUnshallow{}
-	case CheckoutPRMergeBranchMethod, CheckoutPRManualMergeMethod, CheckoutForkPRManualMergeMethod, CheckoutPRDiffFileMethod:
+	case CheckoutPRMergeBranchMethod, CheckoutPRManualMergeMethod, CheckoutPRDiffFileMethod:
```

Comment:
do not we want to add fallback strategy for the 2 new checkout methods?

Reply:
Fixed.